lib/elixir/lib/float.ex

```diff
@@ -380,6 +380,7 @@ defmodule Float do
 
   """
   @since "1.4.0"
+  @spec ratio(float) :: {pos_integer, pos_integer} | {neg_integer, pos_integer}
```

Comment:
`{pos_integer | neg_integer, pos_integer}` would be better then?

Reply:
Huh, didn't know you could do that. Totally better.